dex/testing/firo/README_ELECTRUM_HARNESSES.md

```diff
@@ -0,0 +1,130 @@
+
+# Electrum-Firo Harness Support
+
+Harnesses to run Electrum-Firo wallet on regtest.
+
+## 1. Firo Chain Server Test Harness
+
+The **harness.sh** chain server harness is a collection of tmux scripts that collectively
+creates a sandboxed environment for testing dex swap transactions.
+See Also: README_HARNESS.md
+
+## 2. ElectrumX-Firo Test Harness
+
+The harness is a script named **electrumx.sh** which downloads a git repo containing
+release version of ElectrumX-Firo server.
+
+It requires **harness.sh** Firo chain server harness running
+
+## 3. Electrum-Firo Test Harness
+
+The harness is a script named **electrum.sh** which downloads a git repo containg
+release version of Electrum-Firo wallet client.
+
+It requires **electrumx.sh** Firo ElectrumX-Firo server harness running
+
+Which in turn requires **harness.sh** Firo chain server harness running
+
+## Dependencies
+
+The **harness.sh** script depends on [firod] and [firo-cli] to run.
+https://github.com/firoorg/firo/releases binaries or build from source
+
+The **electrumx.sh** script depends on [python3], [python3 pip] and [git] to run.
+Python3 v3.6 is coded but this script was tested using python3.10. Some testing 
+(minimal) was done with 3.7. Git should be latest. Pip will be downloaded and 
+veriosn upgraded by the script into a virtual environment each time.
+
+The **electrum.sh** script depends on [python3], [python3 pip] and [git] to run.
+Python3 v3.6 is coded but this script was tested using python3.10. Some testing 
+(minimal) was done with 3.7. Git should be latest. Pip will be re-downloaded and 
+version upgraded by the script into a virtual environment each time.
+
```

Comment:
I can confirm it does not work on 3.11 without a lot of hacking so can maybe note not to use.

Reply:
Updated, thanks